fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java

```diff
@@ -183,12 +192,26 @@ public CreateTableStmt buildAnalysisJobTblStmt() throws UserException {
         columnDefs.add(new ColumnDef("tbl_name", TypeDef.createVarchar(1024)));
         columnDefs.add(new ColumnDef("col_name", TypeDef.createVarchar(1024)));
         columnDefs.add(new ColumnDef("index_id", TypeDef.create(PrimitiveType.BIGINT)));
+        columnDefs.add(new ColumnDef("col_partitions", TypeDef.createVarchar(ScalarType.MAX_VARCHAR_LENGTH)));
         columnDefs.add(new ColumnDef("job_type", TypeDef.createVarchar(32)));
         columnDefs.add(new ColumnDef("analysis_type", TypeDef.createVarchar(32)));
-        columnDefs.add(new ColumnDef("message", TypeDef.createVarchar(1024)));
-        columnDefs.add(new ColumnDef("last_exec_time_in_ms", TypeDef.create(PrimitiveType.BIGINT)));
-        columnDefs.add(new ColumnDef("state", TypeDef.createVarchar(32)));
+        columnDefs.add(new ColumnDef("analysis_mode", TypeDef.createVarchar(32)));
+        columnDefs.add(new ColumnDef("analysis_method", TypeDef.createVarchar(32)));
         columnDefs.add(new ColumnDef("schedule_type", TypeDef.createVarchar(32)));
+        columnDefs.add(new ColumnDef("state", TypeDef.createVarchar(32)));
+        columnDefs.add(new ColumnDef("sample_percent", TypeDef.create(PrimitiveType.BIGINT)));
+        columnDefs.add(new ColumnDef("sample_rows", TypeDef.create(PrimitiveType.BIGINT)));
```

Comment:
I think put the original analyze SQL here is much more direcly.

Reply:
just for now, will change it later